indico/modules/users/templates/_category.html

```diff
@@ -17,25 +17,27 @@
 {% endmacro %}
 
 {% macro _category_item(category, truncated_path, extra_info='') %}
-    <li class="category-box">
-        <div>
-            <span class="category-box-info">
-                <a href="{{ category.url }}">{{ category.title }}</a>
-                {{ _category_path(category, truncated_path) }}
-                {{ caller() }}
-            {# The caller must provide the closing span tag (</span>). #}
-            {# TODO This should be changed once Jinja is updated to 2.8 with support for set block. #}
-        </div>
+    <li class="category-box flexrow f-a-center">
+        <span class="category-box-info">
+            <a href="{{ category.url }}">{{ category.title }}</a>
+            {{ _category_path(category, truncated_path) }}
+            {{ caller() }}
+        {# The caller must provide the closing span tag (</span>). #}
+        {# TODO This should be changed once Jinja is updated to 2.8 with support for set block. #}
```

Comment:
Perhaps we could take the opportunity to refactor this?

Reply:
Sure! I also just noticed that `suggested_category` renders a div inside a span.. isn't that actually invalid?

Reply:
> Sure! I also just noticed that `suggested_category` renders a div inside a span.. isn't that actually invalid?

Yes, that's not really inline. I've updated both!